ExampleCode/C++/API-Examples/README.md

```diff
@@ -0,0 +1,11 @@
+API Method Examples
+===================
+
+These cpp files are examples of the various methods of the iSENSE C++ Class.
+
+Use the provided Makefile to compile them, try them out and view the source code.
+
+Note - you will need to use a Unix like OS (Linux or Mac OS), have installed cURL and libcurl
+(installed by default by Mac OS!) and also have run the install script which pulls down the
```

Comment:
Yeah that's a good idea - the Teaching/ExampleCode/C++/Projects/ directory has a README that details  that setup, so I'll add a link to that directory.